napari/_app_model/utils.py

```diff
@@ -0,0 +1,90 @@
+from typing import Union
+
+from app_model.expressions import parse_expression
+from app_model.types import Action, MenuItem, SubmenuItem
+
+from napari._app_model import get_app
+from napari._app_model.constants import MenuGroup, MenuId
+
+MenuOrSubmenu = Union[MenuItem, SubmenuItem]
+
+
+def contains_dummy_action(menu_items: list[MenuOrSubmenu]) -> bool:
+    """Returns True if one of the menu_items is the dummy action, otherwise False.
+
+    Parameters
+    ----------
+    menu_items : list[MenuOrSubmenu]
+        menu items belonging to a given menu
+
+    Returns
+    -------
+    bool
+        True if menu_items contains dummy item otherewise false
```

Comment:
Fit in 80c, use imperative voice or value rather than description ([PEP-257](https://peps.python.org/pep-0257/)), fix typo

```suggestion
    """True if one of the menu_items is the dummy action, otherwise False.

    Parameters
    ----------
    menu_items : list[MenuOrSubmenu]
        menu items belonging to a given menu

    Returns
    -------
    bool
        True if menu_items contains dummy item otherwise false
```